lib/resources/r4/related_person.yaml

```diff
@@ -75,4 +76,313 @@ fields:
+  description: Patient that the individual is related to for the encounter.
+  example: |
+    {
+      "patient": {
+        "reference": "Patient/1626016"
+      }
+    }
+
+- name: name
+  required: 'Yes'
+  type: List of HumanName
+  description: The name associated with the individual.
+  action: create
+  example: |
+    {
+      "name": [
+        {
+          "use": "official",
+          "family": "Doe",
+          "given": [
+            "John",
+            "Smith"
+          ],
+          "period": {
+            "start": "2010-05-17T14:54:31.000Z"
+          }
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li> Only one name <b>must</b> be provided. The provided name must meet the following conditions:
+        <ul>
+          <li> Must have a `use` of 'official'. </li>
+          <li> Must not have an `end` date on `period`, or must not include `period`. </li>
+          <li> Must include both `given` and `family`. </li>
+          <li> Must contain either the `given` or `family` fields. </li>
+          <li> Cannot contain the `text` field. </li>
+          <li> Up to two `given` names may be provided. If additional given names are required they must be provided in the second value as a single string separated by blank spaces. </li>
+          <li> Only one `prefix` and one `suffix` may be provided. </li>
+          <li> When specifying a `period`, the fields must include a time component with a timezone. </li>
+        </ul>
+      </li>
+    </ul>
+  url: https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.name
+
+- name: Related Person Encounter
+  required: 'Yes'
+  type: Extension
+  description: A reference to an Encounter.
+  action: create
+  note: |
+    <ul>
+      <li>The <code>valueReference.reference</code> must be an Encounter reference.</li>
+    </ul>
+  example: |
+    {
+      "extension": [
+          {
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/related-person-encounter",
+            "valueReference": {
+                "reference": "Encounter/2467924"
+            }
+          }
+      ]
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/related-person-encounter?_format=json
+
+- name: Relationship Level
+  required: 'No'
+  type: Extension
+  description: The resource's relationship to either the patient or encounter level.
+  action: create
+  note: |
+    <ul>
+      <li>If provided, the <code>valueCodeableConcept.coding[0].code</code> must be `Encounter`.</li>
+    </ul>
+  example: |
+    {
+      "extension": [
+          {
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/relationship-level",
+                "valueCodeableConcept": {
+                    "coding": [
+                      {
+                          "system": "http://hl7.org/fhir/resource-types",
+                          "code": "Encounter",
+                      }
+                    ]
+                }
+          }
+      ]
```

Comment:
nit: formatting is off.

Reply:
updated here: https://github.com/cerner/fhir.cerner.com/commit/c5a51de325f428be24ed8e8c510ddcd737ea96c3#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R202-R212

Reply:
The coding section still has a few issues.

Reply:
updated here: https://github.com/cerner/fhir.cerner.com/commit/5803ca15b4740206577992e4a5177fece84ff0fb#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R205-R210